src/makieplotting.jl

```diff
@@ -33,7 +33,7 @@ keyword arguments are:
     deformation_field=:default,
     process=postprocess,
     colormap=:cividis,
-    colorrange=(0,1),
+    colorrange=(0,0),
```

Comment:
I think what you want is something like

https://github.com/MakieOrg/Makie.jl/blob/f8c22138683c25b02fa9a6f89221a81b0e62f6a5/src/stats/hexbin.jl#L25C26-L25C35

https://github.com/MakieOrg/Makie.jl/blob/f8c22138683c25b02fa9a6f89221a81b0e62f6a5/src/stats/hexbin.jl#L181-L199

do I understand correctly?

Reply:
That might work too - I don't know the internal details enough to say, just made the minimum possible intrusive change I could come up with. 
Currently, the problem was just that the provided colorrange was just overwritten [here](https://github.com/Ferrite-FEM/FerriteViz.jl/blob/0daec77134a53300dcdd31e9472e073830c86431/src/makieplotting.jl#L69)

So any solution respecting the provided input would be good.